versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application.mdx

```diff
@@ -24,7 +24,7 @@ see [Installing Istio](/versioned_docs/version-0.3.0/getting-started-with-cloud-
 - You have the slice created across worker clusters. For more information, 
 see [Creating a Slice](/versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/installing-kubeslice/creating-a-slice.mdx).
 
-## Creating the Bookinfo Deployment YAML Files
+## Creating the BookInfo Deployment YAML Files
```

Comment:
As per https://istio.io/latest/docs/examples/bookinfo/, it is `Bookinfo`

Reply:
updated